pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -266,6 +274,30 @@ func ingressBackendToHTTPRoute(backend *v1beta1.IngressBackend, namespace string
 	}
 }
 
+func resolveNamedPort(backend *v1beta1.IngressBackend, namespace string, client kubernetes.Interface) int32 {
+	svc, err := client.CoreV1().Services(namespace).Get(context.TODO(), backend.ServiceName, metaV1.GetOptions{})
+	if err != nil {
+		return -1
```

Comment:
return `(int, error)` instead of magic numbers

Reply:
Done